### ISSUE TITLE
How to perform model validation

### ISSUE THREAD

**Issue by doublehops** [amranidev/scaffold-interface#150]
I created a new table/model/etc.. with amranidev scaffolding but I can't see how to handle any validation rules. It doesn't seem to add validation code automatically and I can't find any mention of it.

**Comment by amranidev** [comment]
@doublehops that is correct!!, you can do that manually after creating a new table, so you're free to do that however you like!!

Thanks For submitting this issue, if you have any idea about how we could add this feature, feel free to submit your pull request!!
